<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_manage_deliveryman</class>
 <widget class="QWidget" name="admin_manage_deliveryman">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理配送员信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
                QLabel {
                    color: #333;
                    font: 14px &quot;幼圆&quot;;
                    font-weight: bold;
                }

                QLineEdit,
                QComboBox  {
                    padding: 8px;
                    font-size: 14px;
                    border: 1px solid #ccc;
                    border-radius: 4px;
                }

                QTableWidget {
                    font: 14px &quot;幼圆&quot;;
                    background-color: #cdefff;
                }

                QTableWidget::item {
                    font: 14px &quot;幼圆&quot;;
                }

                QHeaderView {
                    font: bold 16px &quot;幼圆&quot;;
                }
                
                QTableWidget::item:selected {
                    background-color: #a6a6a6;
                }

                QTableWidget QHeaderView::section {
                    background-color: #65a3ff;
                    color: white;
                    font-weight: bold;
                    font-size: 16px;
                    padding: 2px;
                }

                QTableWidget::item {
                    padding: 10px;
                }

                QPushButton {
                    padding: 8px 16px;
                    font-size: 15px;
                    font-weight: bold;
                    border: 1px solid #4CAF50;
                    border-radius: 4px;
                    color: #fff;
                    background-color: #4CAF50;
                }
                
                QPushButton:hover {
                    background-color: #45a049;
                    border: 1px solid #45a049;
                }

                QTextEdit {
                    font-size: 14px;
                    border: 1px solid #ccc;
                    border-radius: 4px;
                }

                QScrollBar:vertical {
                    border: 1px solid #ccc;
                    background: #f0f0f0;
                    width: 10px;
                    margin: 0px 0px 0px 0px;
                }

                QScrollBar::handle:vertical {
                    background: #4CAF50;
                    min-height: 20px;
                }
            </string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>991</width>
     <height>461</height>
    </rect>
   </property>
   <property name="title">
    <string>配送员信息</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>921</width>
      <height>341</height>
     </rect>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>工号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>密码</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>姓名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>电话</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>工作点1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>工作点2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>当前位置</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>年龄</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>当前状态</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="returnBtn">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>10</y>
      <width>121</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>返回上一级</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>631</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>工号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="workIdInputTop"/>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>工作点：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="workCityBox">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchBtn">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>991</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>选择表单中的员工信息在下方修改</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>821</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="createBtn">
       <property name="text">
        <string>新建员工信息</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="updBtn">
       <property name="text">
        <string>修改员工信息</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="delBtn">
       <property name="text">
        <string>删除员工信息</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>821</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>工号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="workIdInputBottom">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pwdInput"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>821</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameInput"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>电话：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="telInput"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>年龄：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ageInput"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>821</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>工作点1：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="workCityInput1"/>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>工作点2：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="workCityInput2"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
